### ISSUE TITLE
Themes not listed on the registry

### ISSUE THREAD

**Issue by noraj1337** [jsonresume/theme-manager#40]
I made a traduction in french of Keloran theme and published it:
+ github: https://github.com/noraj1337/jsonresume-theme-keloran-fr
+ npmjs: https://www.npmjs.com/package/jsonresume-theme-keloran-fr

A few days after it was still not listed on http://registry.jsonresume.org/.

So I just went to http://themes.jsonresume.org/keloran-fr and it worked like a charm.

Then I went back to http://registry.jsonresume.org/ and this time it was listed.

So I suppose that theme are not listed under http://registry.jsonresume.org/ until http://themes.jsonresume.org/nameOfTheTheme was reached at least one time.

Am I right? Is it normal? Can you fix this behaviour or document it on the README page if it is done on purpose.

**Comment by thomasdavis** [comment]
You are correct. To be honest, I will probably be taking down the resume editor on the root of that domain anyway.
